crates/matrix-sdk/src/event_cache/store.rs

```diff
@@ -18,6 +18,9 @@ use matrix_sdk_common::deserialized_responses::SyncTimelineEvent;
 
 use super::linked_chunk::{Chunk, ChunkIdentifier, Error, Iter, LinkedChunk, Position};
 
+/// An alias for the real event type.
+pub type Event = SyncTimelineEvent;
```

Comment:
I wonder if that won't confuse people more than it will help.

Reply:
No idea if that will help, people complained about too many similarly named types in the internal as well as in the external API.

Reply:
I prefer to keep this, and wait for people to complain… It's a rather isolated API.

Reply:
Yup, it's `pub(crate)` now.